src/DynamoCoreWpf/Views/Core/DynamoView.xaml

```diff
@@ -637,7 +637,7 @@
                             </MenuItem>
                             <MenuItem x:Name="PackagesMenuGuide"
                                       Focusable="False"     
-                                      
+                                      Visibility="Collapsed"
```

Comment:
@filipeotero Is it because there is no way to come out of it yet? If not, we dont have to hide it for now because of the nature of DynamoSandbox, or if you prefer we can hide it behind a debug mode or display only on Debug builds, etc

Reply:
It's because we just have the first step. I can leave it visible if it's a not problem.

Reply:
I see. Thanks for the consideration, for sandbox, given this is a new feature, it only become a problem, e.g. when 2.13.0 is released, this feature has been completed. In that case, we will need to consider hiding under a condition. Please flip the visibility flag.